services/order.py

```diff
@@ -0,0 +1,28 @@
+from django.db import transaction
+
+from db.models import Order, User, Ticket
+
+
+def create_order(tickets: list,
+                 username: str,
+                 date: str = None
+                 ) -> None:
+    with transaction.atomic():
```

Comment:
Use `transaction.atomic` as a decorator instead of a context manager. This makes your code more readable

Reply:
I did used:)

Reply:
oh in order also